packages/create-corvid-app/README.md

```diff
@@ -0,0 +1,14 @@
+# Create Corvid App
+
+`create-corvid-app` allows you to do the necessary setup for using `corvid-cli` and download your [Wix](https://www.wix.com) site in just one command.
```

Comment:
I would change to: "`create-corvid-app` sets up a local site project to use the `corvid-cli` and clones your [Wix](https://www.wix.com) site in a single command."